roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -59,6 +59,16 @@
   failed_when: "'already exists' not in osmiq_perm_task.stderr and osmiq_perm_task.rc != 0"
   changed_when: osmiq_perm_task.rc == 0
 
+- name: Configure 3_2 role/user permissions
+  command: >
+    {{ openshift.common.admin_binary }} {{item}}
+    --config={{manage_iq_tmp_conf}}
+  with_items: "{{manage_iq_openshift_3_2_tasks}}"
+  register: osmiq_perm_3_2_task
+  failed_when: osmiq_perm_3_2_task.rc != 0
+  changed_when: osmiq_perm_3_2_task.rc == 0
```

Comment:
@detiber @deads2k The above is wrong, but I could not think of anything better:
oadm policy does not tell me if the role existed before (exit code for both cases is 0 and no output)
The only thing I could think of is parsing the result of 
```bash
oc describe clusterPolicyBindings :default
```
but that does not feel right